### PR TITLE
[Issue 7032][Doc] Update connector admin CLI for suporting DLQ for sink/source.

### LINKED ISSUE #7032: Pulsar supporting DLQ for sources/sinks
Currently Pulsar doesn't support DLQ for sources and sinks. Only functions support DLQ.
It is very important for sinks like ElasticSearch which is widely used in production env for logging systems to be able to redirect unacknowledged messages to another topic , keeping otherwise the brokers running as normal. 

I would like to see a setting in consumer.conf or broker.conf that enables DLQ on a Pulsar configuration, so DLQ will be turned on for sources, sinks or any functions. 

or

I would like to create a sink with an extra flag --DLQ <topic> parameter. Giving a topic to the parameter will turn on DLQ.

### PR DESCRIPTION
**Pull request by Huanli-Meng**
Fix #7032 

Update doc for issue #7032 based on the PR #7094 

### Motivation

1. Update doc for issue #7032 for supporting DLQ for sink/source.

2: fix legacy link error for `reload` command in "How to use Pulsar connectors" document

### Modifications

1: add options in localrun for sink connector in connector admin CLI.

2: fix legacy link error for `reload` command in "How to use Pulsar connectors" document

### CODE REVIEW

**Comment by Huanli-Meng** [comment]
@Jennifer88huang @jiazhai , could you please help review? Thanks.

File: site2/docs/io-cli.md [hunk 1]
```diff
@@ -557,7 +557,9 @@ $ pulsar-admin sinks localrun options
 | `--client-auth-plugin` | Client authentication plugin using which function-process can connect to broker.
 |`--cpu`|The CPU (in cores) that needs to be allocated per sink instance (applicable only to the Docker runtime).
 | `--custom-schema-inputs` | The map of input topics to Schema types or class names (as a JSON string).
-| `--custom-serde-inputs` | The map of input topics to SerDe class names (as a JSON string).
+| `--max-redeliver-count` | Maximum number of times that a message is redelivered before being sent to the dead letter queue.
+| --dead-letter-topic | Name of the dead letter topic where the failing messages are sent.
```
Comment (jiazhai) [inline site2/docs/io-cli.md:561]: `--dead-letter-topic ` add "``"

File: site2/docs/io-cli.md [hunk 2]
```diff
@@ -557,7 +557,9 @@ $ pulsar-admin sinks localrun options
...
+|| `--custom-serde-inputs` | The map of input topics to SerDe class names (as a JSON string).
```
Comment (jiazhai) [inline site2/docs/io-cli.md:562]: `||` => `|`

Comment (Huanli-Meng) [reply site2/docs/io-cli.md]: Thanks for your comments, the doc is updated.

File: site2/docs/io-cli.md (hunk 1 above)
Comment (Huanli-Meng) [reply site2/docs/io-cli.md]: Thanks for your comments. The doc is updated.

File: site2/docs/io-cli.md [hunk 3]
```diff
@@ -557,6 +557,8 @@ $ pulsar-admin sinks localrun options
 | `--client-auth-plugin` | Client authentication plugin using which function-process can connect to broker.
 |`--cpu`|The CPU (in cores) that needs to be allocated per sink instance (applicable only to the Docker runtime).
 | `--custom-schema-inputs` | The map of input topics to Schema types or class names (as a JSON string).
+| `--max-redeliver-count` | Maximum number of times that a message is redelivered before being sent to the dead letter queue.
+| `--dead-letter-topic` | Name of the dead letter topic where the failing messages are sent.
```
Comment (Anonymitaet) [inline site2/docs/io-cli.md:561]: failing → failed ?

**Comment by ADHB** [comment]
Hi guys. Thanks for the good work. 
When I use Elastic sink with dead_letter_topic i get:
[root@clstpulsar01 apache-pulsar-2.6.1]# bin/pulsar-admin sink create --name elastic --dead_letter_topic test-tenant/test-namespace/elastic_dlq --sink-config-file conf/elastic_sink.yml --tenant test-tenant --namespace test-namespace --inputs netty  --archive connectors/pulsar-io-elastic-search-2.6.1.nar
Unknown option: --dead_letter_topic

Any thoughts?

**Pull request merged**